### ISSUE TITLE
Menu should have option to set order

### ISSUE THREAD

**Issue by Light-Wizzard** [cutelyst/CMlyst#13]
Currently when I create a menu; it reorders it, I would like to have my Menu look like this: Home About,
this is not possible the way its written.  

It would be nice to be able to set the order and change it.

Thanks

**Comment by dantti** [comment]
Yeah, sadly that part is a bit broken not well done, if you can provide patches I'm glad to review.
